containers/build

```diff
@@ -13,4 +13,4 @@ podman run -ti \
     -v $TARGET:/build \
     -v $HOME/.local/share/containers:/var/lib/containers \
     quay.io/buildah/stable \
-    buildah -t testenv bud /build
+    buildah -t testenv bud /build
```

Comment:
Was the newline removed on purpose?

Reply:
Yeah, it was. The empty new line caused the crash of the script.